README.md

```diff
@@ -71,7 +71,7 @@ Supported Compilers
 This library uses C++14 (or newer when available).
 
 |Accelerator Back-end|gcc 5.5 <br/> (Linux)|gcc 6.4/7.3 <br/> (Linux)|gcc 8.1 <br/> (Linux)|gcc 9.1 <br/> (Linux)|gcc 10.1 <br/> (Linux)|clang 4 <br/> (Linux)|clang 5 <br/> (Linux)|clang 6 <br/> (Linux)|clang 7 <br/> (Linux)|clang 8 <br/> (Linux)|clang 9 <br/> (Linux)|clang 10 <br/> (Linux)|Apple LLVM 11.2.1-12.2.0 <br/> (macOS)|MSVC 2019 <br/> (Windows)|
-|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
```

Comment:
since you added a column there is now a missing column entry for HIP. When you review this PR you can clock top right to "Display rich diff` to see it rendered.

Reply:
Good catch! And the "Display rich diff" is super useful, thx!